tests/verify.sh

```diff
@@ -3,6 +3,10 @@
 set +e
 set -o pipefail
 
+source ../../ods-configuration/ods-core.env
```

Comment:
I get on my bash on ubuntu (via windows WSL):

```
Verifying ODS installation
../../ods-configuration/ods-core.env: line 248: syntax error near unexpected token `('
../../ods-configuration/ods-core.env: line 248: `PROV_APP_MAIL_PASSWORD=ENC(...)'
```

in ENC(...) the only special chars are + and =

Any idea?

Reply:
@michaelsauter  - maybe you have an idea? .. this only happens on @renedupont 's shell ..

Reply:
That issue is not shell related. But the password may contain chars which get interpreted on the shell. Enclosing in `""` helps - e.g. ` PROV_APP_MAIL_PASSWORD="....."`